SecretSalty/src/main/java/com/secretsalty/secretsalty/endpoint/controller/UserController.java

```diff
@@ -39,6 +39,7 @@ public UserController(UserRepository userRepository, FirebaseAuth firebaseAuth)
   @PostMapping()
   @ResponseStatus(HttpStatus.CREATED)
   public User create(@RequestBody User newUser) {
+    System.out.println("creating new user: " + newUser);
```

Comment:
if we want to log stuff (temporarily, even) we should use a logger
-> here I'd suggest to simply use the @log4j annotation & put a log.debug("...", newUser) into this.